### DISCUSSION TITLE
Turn Cortana into Copilot, make the no longer working Cortana button in the Windows 10 taskbar function as the Copilot button instead!

### DISCUSSION THREAD

**Discussion by zake2002v2** [valinet/ExplorerPatcher#2948] [Ideas]
If you guys are planning to integrate the Windows 10 Copilot button [located where the notification button should be](https://github.com/valinet/ExplorerPatcher/assets/42307974/41a6f42b-03ff-4965-bab9-4815f4d6f8c2), I may like to suggest that it can **optionally instead replace the discontinued Cortana button**, so that calling out Cortana will instead bring out Copilot, since Cortana is discontinued and no longer supported.

**Comment by haydengriz** [comment]
This was already suggested and is planned
